apps/src/hamburger/hamburger.js

```diff
@@ -7,18 +7,29 @@ import {
 
 export const initHamburger = function() {
   $(function() {
-    $('#hamburger-icon').click(function(e) {
-      $(this).toggleClass('active');
-      $('#hamburger #hamburger-contents').slideToggle();
-      e.preventDefault();
+    $('#hamburger').on('keypress click', function(e) {
+      if (
+        (e.type === 'keypress' &&
+          e.which === 13 &&
+          e.target.className !== 'hamburger-expandable-item item') ||
+        e.type === 'click'
+      ) {
```

Comment:
is there not a more generalizable way to do this? I'd imagine if we use an element that's semantically intended to be interactable, some of this work might be done for us

Reply:
I agree that we should use a different element for this - we discussed a bit about this when we were planning this hackathon work and decided that updating this to a different element is out of the scope of this hackathon project. We have a lot of clickable divs in our repo that we should generally move away from and transition to interactive elements, specifically, lots of our "buttons" are not button elements. I'm hopeful that this work can be built into our future 'component library'. In particular, this case is somewhat odd because we have what looks like a button, but it's a menu. I'm not sure what the best element would be to use here that would change the style small enough that we wouldn't need product or design approval. For what it's worth, we have another PR coming up that does switch out some navigation clickable-div-buttons to button-element-buttons.

Reply:
Although, now that I've said that, this seems like it is similar to what we might want in this particular case: https://www.w3schools.com/howto/howto_js_dropdown.asp

Reply:
> decided that updating this to a different element is out of the scope of this hackathon project

That does make sense, and I'm glad y'all are considering these implications for the long term!

I would guess, though, that updating this to use a button like in the example you linked would be easier even in the short term than trying to manually implement all the built-in accessibility considerations like we're trying to do here